web-api/src/server.rs

```diff
@@ -91,6 +92,7 @@ where
                 .wrap_fn(wrap_non_json_errors)
                 .wrap_fn(tracing_log_request)
                 .wrap(middleware::Compress::default())
+                .wrap(Cors::permissive())
```

Comment:
:pin: for now good enough, but I guess in the future we want to only support the domains of the specific customer probably by having a config entry with a list of allowed domains.